src/app/public-pages/public-profile/public-profile.component.html

```diff
@@ -32,7 +33,18 @@
 
           <div *ngIf="profile?.photo && profile.photo.avatar" [ngStyle]="{ 'background-image': 'url(' + profile?.photo.avatar + ')' }" class="photo-container"></div>
 
-          <h1>{{hostname}}</h1>
+          <div class="public-profile-profile-of">PUBLIC PROFILE OF:</div>
```

Comment:
More semantic approach would be to use standard letter cases and then add CSS rule `text-transform: uppercase`